File: C++/Random Number Guessing Game.cpp
#include <iostream>
#include <ctime>

using namespace std;

int main() 
{

  int num;
  int guess;
  int tries = 0;
  
srand(time(NULL));
  num = (rand() % 100) + 1;

  cout << "******* GUESSING THE NUMBER GAME *******" << '\n' << '\n';

  do{
    
    cout << "Enter a guess (1-100): ";
    cin >> guess;
    tries++;

    if(guess > num){
      cout << "Too high" << '\n';
    }
    else if(guess < num){
      cout << "Too low" << '\n';
    }
    else{
      cout << "\nCorrect! You got it in " << tries << " tries!" << '\n' << '\n';
    }
  }while(guess != num);

  cout << "**************************************";
  
  return 0;
}

File: Python/Random Number Guessing Game.py
import random
import os

os.system('cls')

rand_num = random.randint(1, 100)

guess = None
no_guess = 0

print("******* GUESSING THE NUMBER GAME *******\n")

while guess != rand_num:
    guess = input("Guess the number(1-100): \n")

    guess = int(guess)
    
    if guess < rand_num:
        print("Too low")
        no_guess = no_guess + 1
        continue

    if guess > rand_num:
        print("Too high")
        no_guess = no_guess + 1
        continue
    
print("\nCorrect! You guessed it in" , no_guess, "tries\n")

print("**************************************")
